roles/openshift_descheduler/defaults/main.yaml

```diff
@@ -0,0 +1,39 @@
+---
+# descheduler common setup
+openshift_descheduler_state: present
+openshift_descheduler_namespace: openshift-infra
+openshift_descheduler_tmp_location: /tmp
+openshift_descheduler_delete_config: True
+
+# descheduler service account setup
+openshift_descheduler_service_account: descheduler-sa
+
+# descheduler cluster role setup
+openshift_descheduler_cluster_role_name: descheduler-cr
+
+# descheduler policy setup
+# by default only RemoveDuplicates strategy is enabled
+openshift_descheduler_policy_configmap_name: descheduler-policy-configmap
+openshift_descheduler_strategy_remove_duplicates_enabled: true
+openshift_descheduler_strategy_remove_pods_violating_interpod_antiaffinity_enabled: false
```

Comment:
It is a long name for a variable. What about to do:
```yaml
openshift_descheduler_strategies:
- strategy: remove_duplicates
  enabled: true
- strategy: remove_pods_violating_interpod_antiaffinity
  enabled: false
- strategy: low_node_utilization
  enabled: false
  cpu_threshold: 40
  mem_threshold: 40
  pods_threshold: 40
  cpu_target_threshold: 70
  mem_target_threshold: 70
  pods_target_threshold: 70
```
instead?

Or maybe:
```yaml
openshift_descheduler_strategies:
  remove_duplicates:
    enabled: true
  remove_pods_violating_interpod_antiaffinity:
    enabled: false
  low_node_utilization:
    enabled: false
    cpu_threshold: 40
    mem_threshold: 40
    pods_threshold: 40
    cpu_target_threshold: 70
    mem_target_threshold: 70
    pods_target_threshold: 70
```
so it is easier to traverse.

Reply:
I am not sure I understand what you are proposing or if this would work, but these are default values not a config structure.  These default values are then fed to populate template file policy.yaml.j2.

Reply:
The variable names are too long which makes the code harder to read. The templating in the `policy.yaml.j2` can be updated to reflect the `openshift_descheduler_strategies` variable is a dictionary. The point is to simplify the variables needed to setup the descheduler.

Reply:
I agree but I will have to learn how to do that in ansible :-) that's why I was saying I don't understand what you are proposing. Anyway I will look into that.